src/mobility/components/OperatorBenefit.tsx

```diff
@@ -37,14 +36,18 @@ export const OperatorBenefit = ({
       <Section>
         <GenericSectionItem>
           <View style={style.benefitContainer}>
-            <View style={style.benefitImage}>
               {isUserEligible && benefit.imageWhenActive && (
-                <SvgXml xml={benefit.imageWhenActive} />
+                <Image
+                  style={style.benefitImage as ImageStyle}
+                  source={{uri: benefit.imageWhenActive}}
+                />
               )}
               {!isUserEligible && benefit.imageWhenNotActive && (
-                <SvgXml xml={benefit.imageWhenNotActive} />
+                <Image
+                  style={style.benefitImage as ImageStyle}
+                  source={{uri: benefit.imageWhenNotActive}}
+                />
               )}
-            </View>
             <View style={style.benefitContent}>
               <ThemeText type="body__primary--bold">{heading}</ThemeText>
```

Comment:
```suggestion
              {heading && (
                <ThemeText type="body__primary--bold">{heading}</ThemeText>
              )}
```

This change will fix the issue where ThemeText takes some room even when blank, causing unwanted whitespace:

| Before | After |
|--------|-------|
| <img width="200" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/65b1a88b-45e5-402f-addd-8abea56d82b1" /> | <img width="200" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/18a8592e-57b4-4bf1-b769-42f4fb1e4056" /> |

Reply:
updated